ksql-cli/src/main/java/io/confluent/ksql/cli/Cli.java

```diff
@@ -118,10 +122,14 @@ private static void validateClient(
     try {
       RestResponse restResponse = restClient.makeRootRequest();
       if (restResponse.isErroneous()) {
+        KsqlErrorMessage ksqlError = restResponse.getErrorMessage();
+        if ((ksqlError.getErrorCode() / Errors.HTTP_TO_ERROR_CODE_MULTIPLIER)
+            == NOT_ACCEPTABLE.getStatusCode()) {
+          writer.format("Current CLI version no longer supported: %s\n\n", ksqlError);
```

Comment:
Consider "_This_ CLI version no longer supported".

It would also be really cool if we could include a link for where to download a new CLI from. Of course, we'd then need to know which CLI they should download, which would depend on which server version they have.

Reply:
Maybe we can include the link in the error returned by the server if/when we do stop serving an api version.